salt/modules/mac_xattr.py

```diff
@@ -47,7 +47,7 @@ def list(path, hex=False):
     cmd = 'xattr "{0}"'.format(path)
     ret = __salt__['cmd.run'](cmd)
 
-    if "No such file" in ret:
+    if "No such file" in ret or not ret:
         return None
```

Comment:
Please don't return None here. I think @rallytime has already discussed this with you in another context, so I'm guessing you'll come around to this one.